backend/src/index.js

```diff
@@ -1,9 +1,12 @@
 import express from 'express';
+import initDb from './db/mongoose';
+import userRouter from './routers/userRouter';
 
 const app = express();
-console.log('es6 features works');
-app.get('/', (_req, res) => {
-  res.send('hello world');
-});
+
+initDb();
+
+app.use(express.json());
+app.use(userRouter);
 
 app.listen(3000, () => console.log('App listening on port 3000!'));
```

Comment:
Please wait for initializing db before start listening to the app.

Reply:
Cool, will do it.